### PR TITLE
Create Ordering.md

### PR DESCRIPTION
**Pull request by pifopi**
Add wiki for dex ordering program

### CODE REVIEW

File: Wiki/Programs/PokemonSwSh/BoxReorderNationalDex.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Box Reorder National Dex
+
+## Program Description
+
+Order boxes of Pokémon, based on their national dex index.
```
Comment (Mysticial) [inline Wiki/Programs/PokemonSwSh/BoxReorderNationalDex.md:5]: Image Kudasai? :)

File: Wiki/Programs/PokemonSwSh/EggFetcherMultiple.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Egg Fetcher Multiple
+
+## Program Description
+
+Fetch eggs into your boxes without hatching them.
+
+This program is used with [EggHatcher](EggHatcher.md) to fetch and hatch eggs. Thus you first fetch a bunch of eggs with EggFetcher, then you hatch them with EggHatcher.
+
+Unlike the other egg fetcher, it will fetch a chosen number of eggs before switching one parent and trying again.
```
Comment (Mysticial) [inline Wiki/Programs/PokemonSwSh/EggFetcherMultiple.md:9]: Image Kudasai? :)

File: Wiki/Programs/PokemonSwSh/BoxReorderNationalDex.md (hunk 1 above)
Comment (pifopi) [reply Wiki/Programs/PokemonSwSh/BoxReorderNationalDex.md:5]: image added ! let me know if this one is good or not !

File: Wiki/Programs/PokemonSwSh/EggFetcherMultiple.md (hunk 2 above)
Comment (pifopi) [reply Wiki/Programs/PokemonSwSh/EggFetcherMultiple.md:9]: I used the egg fetcher 2 image because it is exactly the same location to start the program

**Pull request merged**